contracts/test-contracts/Callable.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity ^0.8.0;
+pragma solidity 0.8.20;
```

Comment:
Can we do 0.8.22? Can be cheaper 
- https://soliditylang.org/blog/2023/10/25/solidity-0.8.22-release-announcement

Reply:
for system contracts we can, i chose 0.8.20 as thats the newest supported by foundry

Reply:
if we're okay with the mismatch between system and era contracts than im all for it